src/mono/wasm/debugger/BrowserDebugProxy/DevToolsHelper.cs

```diff
@@ -66,6 +66,7 @@ internal class DotnetObjectId
         public string Scheme { get; }
         public int Value { get; }
         public int SubValue { get; set; }
+        public ElementType ElementType { get; set; }
```

Comment:
Is the `ElementType` in the `methodId` used for anything other than valuetype/class differentiation? We should just add a bool, or simple `0/1` instead of the elementType to the `methodId`. And then expose a `public bool IsValueType { get; }` property here.

Reply:
It's used only for that, bool will do the job as well.